src/i18n.js

```diff
@@ -0,0 +1,13 @@
+export function i18n(language) {
```

Comment:
@EECOLOR Zouden we de CMS ui niet in het Engels doen? Volgensmij is het nl. niet mogelijk de hele interface te vertalen.